opennms-webapp-rest/src/main/java/org/opennms/web/rest/v1/HeatMapRestService.java

```diff
@@ -168,6 +171,16 @@ public Response outagesByForeignsources() throws IOException {
     @GET
     @Produces(MediaType.APPLICATION_JSON)
     @Transactional
+    @Path("outages/managedServices")
+    public Response outagesByServices() throws IOException {
+        final List<HeatMapElement> heatMapElements = m_outageDao.getHeatMapItemsForEntity("service.servicename", "service.serviceid", null, null);
+        final JSONObject jo = new JSONObject(transformResults(heatMapElements, System.getProperty(SERVICE_FILTER_PROPERTY_KEY, SERVICE_FILTER_PROPERTY_DEFAULT)));
+        return Response.ok(jo.toString(), MediaType.APPLICATION_JSON).build();
```

Comment:
Do you have the same issue here as the one Dustin yesterday fixed?
Does `jo.toString()` raise an exception?